src/app/models/resource.ts

```diff
@@ -37,6 +37,12 @@ export class Resource {
   'attr-hidden'?: boolean;
   'attr-spotlight'?: boolean;
 
+  static isMetaTool(resource: Resource): boolean {
+    var type = resource.resourceType ? resource.resourceType.name : null;
+    if (type == null) type = resource['resource-type'];
```

Comment:
Also, should this be combined with the above line?
```
var type = resource.resourceType ? resource.resourceType.name : resource['resource-type'];
```

Reply:
Third, I think `type` should be a `const` not a `var`.

Reply:
the reason I didn't go with
```
var type = resource.resourceType ? resource.resourceType.name : resource['resource-type'];
```

is it would return null if there is a `resourceType`, but the name itself is null. There is a null-coalescing operator that fits a lot better, but that's only available on newer versions of typescript

Reply:
`==` wasn't intentional, going to update it now

Reply:
I updated this logic